recipes/rsv-typer/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.1" %}
+
+package:
+  name: rsv-typer
+  version: {{ version }}
+
+source:
+  url: https://github.com/DiltheyLab/RSVTyper/archive/refs/heads/main.tar.gz
```

Comment:
The source URL should be stable, preferably pointing to a tagged version, see: https://bioconda.github.io/contributor/guidelines.html#stable-url

Could you please ask the repository owner to make a release on Github that can be pointed to here?

Reply:
Hey, thank you so much for your review! I made a release and changed the URL

Reply:
one small thing, you can use the version variable `{{ version }}` in the source url too. these should also match the release tag in the file name (`0.1` vs `0.1.0`). have a look at other bioconda recipes to see the pattern. once that's fixed i think the recipe is good to go!

Reply:
Thank you! I changed those as well now